inginious/frontend/app.py

```diff
@@ -87,6 +88,8 @@ def _put_configuration_defaults(config):
     config["SESSION_USE_SIGNER"] = True
     config["PERMANENT_SESSION_LIFETIME"] = config['session_parameters']["timeout"]
     config["SECRET_KEY"] = config['session_parameters']["secret_key"]
+    config["CACHE_TYPE"] = "simple"
```

Comment:
This is used by the LTI code to temporarily store data on ongoing OIDC handshakes. I tested it on my instance using WSGI but there might be more dragons, e.g. related to threading